tests/test_serializers.py

```diff
@@ -75,36 +75,66 @@ def test_prettyprint():
         assert '{\n  "key": "1"\n}' == serializer.serialize_object({"key": "1"})
 
 
-def test_marshmallow_serializer_without_context():
+def test_marshmallow_json_serializer_without_context():
     serializer = MarshmallowSerializer(
         format_serializer_cls=JSONSerializer,
         object_schema_cls=UITestSchema,
         list_schema_cls=BaseListSchema,
     )
     test = {"test": _("test")}
-
-    list_test = {"hits": {"hits": [{"test": _("test")}]}}
+    list_test = {"hits": {"hits": [{"test": _("test1")}, {"test": _("test2")}]}}
     assert serializer.serialize_object(test) == '{"title_l10n": "test"}'
     assert (
         serializer.serialize_object_list(list_test)
-        == '{"hits": {"hits": [{"title_l10n": "test"}]}}'
+        == '{"hits": {"hits": [{"title_l10n": "test1"}, {"title_l10n": "test2"}]}}'
     )
 
 
-def test_marshmallow_serializer_with_context():
+def test_marshmallow_json_serializer_with_context():
     serializer = MarshmallowSerializer(
         format_serializer_cls=JSONSerializer,
         object_schema_cls=UITestSchema,
         list_schema_cls=BaseListSchema,
         schema_context={"object_key": "ui"},
     )
     test = {"test": _("test")}
-    list_test = {"hits": {"hits": [{"test": _("test")}]}}
+    list_test = {"hits": {"hits": [{"test": _("test1")}, {"test": _("test2")}]}}
     assert (
         serializer.serialize_object(test)
         == '{"test": "test", "ui": {"title_l10n": "test"}}'
     )
     assert (
         serializer.serialize_object_list(list_test)
-        == '{"hits": {"hits": [{"test": "test", "ui": {"title_l10n": "test"}}]}}'
+        == '{"hits": {"hits": [{"test": "test1", "ui": {"title_l10n": "test1"}}, {"test": "test2", "ui": {"title_l10n": "test2"}}]}}'
+    )
+
+
+def test_marshmallow_csv_serializer_without_context():
+    serializer = MarshmallowSerializer(
+        format_serializer_cls=CSVSerializer,
+        object_schema_cls=UITestSchema,
+        list_schema_cls=BaseListSchema,
+    )
+    test = {"test": _("test")}
+    list_test = {"hits": {"hits": [{"test": _("test1")}, {"test": _("test2")}]}}
+    assert serializer.serialize_object(test) == "\r\n".join(["title_l10n", "test", ""])
+    assert serializer.serialize_object_list(list_test) == "\r\n".join(
+        ["title_l10n", "test1", "test2", ""]
+    )
```

Comment:
minor: would be nice to deserialize and compare to ensure that this is a valid CSV